### PR TITLE
Remove FXIOS-5788 [v112] Remove Explore more from wallpaper onboarding

### PR DESCRIPTION
**Pull request by CatalinNeculaide**
Remove the "Explore more wallpapers" from WallpaperSelectorView

Simulator Screen Shot - iPad Air (5th generation) - 2023-02-23 at 12 43 59

### CODE REVIEW

**Comment by mte-test** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Project coverage: 30.1%</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Edited 1 files</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Created 0 files</td>
    </tr>
  </tbody>
</table>

## Client.app: Coverage: 26.99
| File | Coverage ||
| --- | --- | --- |
WallpaperSelectorViewController.swift | 0.0% | ⚠️

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against 7928a4dedc4bbd83e3bcd47f81ce42b4eb519ef6
</p>

File: Client/Frontend/Home/Wallpapers/v1/UI/WallpaperSelectorViewController.swift [hunk 1]
```diff
@@ -58,14 +58,6 @@ class WallpaperSelectorViewController: WallpaperBaseViewController, Themeable {
         return collectionView
     }()
 
-    private lazy var settingsButton: ResizableButton = .build { button in
-        button.titleLabel?.font = DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .body,
-                                                                                size: 16)
-        button.titleLabel?.textAlignment = .center
-        button.setTitle(.Onboarding.WallpaperSelectorAction, for: .normal)
```
Comment (thatswinnie) [inline Client/Frontend/Home/Wallpapers/v1/UI/WallpaperSelectorViewController.swift:65]: Could you please remove the string from `Strings.swift` as well?

File: Client/Frontend/Home/Wallpapers/v1/UI/WallpaperSelectorViewController.swift [hunk 2]
```diff
@@ -58,14 +58,6 @@ class WallpaperSelectorViewController: WallpaperBaseViewController, Themeable {
...
-        button.accessibilityIdentifier = AccessibilityIdentifiers.Onboarding.Wallpaper.settingsButton
```
Comment (thatswinnie) [inline Client/Frontend/Home/Wallpapers/v1/UI/WallpaperSelectorViewController.swift:66]: We should remove the AccessibilityIdentifier from `AccessibilityIdentifiers.swift` too.

File: Client/Frontend/Home/Wallpapers/v1/UI/WallpaperSelectorViewController.swift (hunk 1 above)
Comment (CatalinNeculaide) [reply Client/Frontend/Home/Wallpapers/v1/UI/WallpaperSelectorViewController.swift:65]: yes, thank you for pointing this out!

File: Client/Frontend/Home/Wallpapers/v1/UI/WallpaperSelectorViewController.swift (hunk 2 above)
Comment (CatalinNeculaide) [reply Client/Frontend/Home/Wallpapers/v1/UI/WallpaperSelectorViewController.swift:66]: thanks for this too

**Review by thatswinnie — APPROVED** [review]

**Review by lmarceau — APPROVED** [review]

**Comment by nbhasin2** [comment]
@Mergifyio backport release/v110.2

**Comment by JohanLorenzo** [comment]
@mergifyio refresh

**Comment by JohanLorenzo** [comment]
@Mergifyio backport release/v111.0

**Comment by JohanLorenzo** [comment]
@Mergifyio backport release/v111.0

**Pull request merged**